sdcm/kcl_thread.py

```diff
@@ -0,0 +1,136 @@
+import os
+import time
+import random
+import logging
+import uuid
+import threading
+
+from sdcm.stress_thread import format_stress_cmd_error, DockerBasedStressThread
+from sdcm.utils.docker_remote import RemoteDocker
+from sdcm.sct_events import StressEvent, InfoEvent, Severity
+from sdcm.cluster import BaseNode
+
+LOGGER = logging.getLogger(__name__)
+
+
+class KclStressEvent(StressEvent):
+    pass
+
+
+class KclStressThread(DockerBasedStressThread):  # pylint: disable=too-many-instance-attributes
+
+    def run(self):
+        _self = super().run()
+        # wait for the KCL thread to create the tables, so the YCSB thread beat this one, and start failing
+        time.sleep(120)
+        return _self
+
+    def build_stress_cmd(self):
+        if self.params.get('alternator_use_dns_routing'):
```

Comment:
can be considered changing to: 
if not target_address := self.params.get('alternator_use_dns_routing'):
    if getattr(self.node_list[0], 'parent_cluster'):
               target_address = self.node_list[0].parent_cluster.get_node().ip_address
     else:
               target_address = self.node_list[0].ip_address